programming/python/libraries.md

```diff
@@ -23,22 +23,23 @@ The following python libraries are installed and available for use in your robot
 * [gpg 1.16.0](https://pypi.org/project/gpg)
 * [iniparse 0.5](https://pypi.org/project/iniparse)
 * [j5 0.13.1](https://pypi.org/project/j5)
-* [j5-zoloto 0.1.0](https://pypi.org/project/j5-zoloto)
+* [j5-zoloto 0.2.0](https://pypi.org/project/j5-zoloto)
 * [kiwisolver 1.3.2](https://pypi.org/project/kiwisolver)
 * [matplotlib 3.4.1](https://pypi.org/project/matplotlib)
 * [numpy 1.21.2](https://pypi.org/project/numpy)
 * opencv 4.5.2
 * [pandas 1.3.3](https://pypi.org/project/pandas)
+* [pillow 8.3.2](https://pypi.org/project/Pillow/)
 * pkg-resources 57.4.0
 * [pydantic 1.8.2](https://pypi.org/project/pydantic)
-* [pygobject 3.40.1](https://pypi.org/project/PyGObject/)
+* [dateutil 2.8.2](https://pypi.org/project/python-dateutil/)
```

Comment:
Looks like dateutil is listed here three times, is that intentional?

Reply:
Have `pygobject` and `pyyaml` been removed from the image (these are what git says used to be on these lines)? If so all looks good to me!